codegen/kuberesource/crd.go

```diff
@@ -15,8 +19,21 @@ import (
 	"k8s.io/utils/pointer"
 )
 
+type Options struct {
+	WithoutSpecHash bool
+}
+
+type Option func(*Options)
+
+func WithoutSpecHash() Option {
```

Comment:
Do we need to represent this option using a function? Could we just pass around a boolean?

Reply:
we could but that would break backwards compatability